pytorch/edgeml_pytorch/graph/rnn.py

```diff
@@ -261,7 +261,7 @@ def __init__(self, input_size, hidden_size, gate_nonlinearity="sigmoid",
         self.zeta = nn.Parameter(self._zetaInit * torch.ones([1, 1]))
         self.nu = nn.Parameter(self._nuInit * torch.ones([1, 1]))
 
-        self.copy_previous_UW()
+        # self.copy_previous_UW()
```

Comment:
Why is this commented out?  This supports sparsity for KWS codes. The fastcell_Example.py doesn't depend on it because this is a bit broken given the new pytorch updates. Check with Harsha.

Reply:
This line is causing segmentation fault in the rnnpool codes

Reply:
Yeah! I know the code is broken, it needs to be fixed. @harsha-simhadri  used this in one of the codes so, we might have to fix it.